README.md

```diff
@@ -2,21 +2,21 @@
 
 The frontend for Codex, written in React with Material UI components. This project is not in production yet, but it is under active development.
 
-## Contributions 
+## Contributions
 
 Found a bug? Submit a report [here](https://github.com/WildMeOrg/codex-frontend/issues/new).
 
-Developer contributions are very much appreciated =). Take a gander at the [contribution guide](https://github.com/WildMeOrg/codex-frontend/blob/master/CONTRIBUTION_GUIDE.md). If you a Java dev looking to help with our project, take a look at the [backend](https://github.com/WildMeOrg/Wildbook). And if you are a Python dev or data scientist take a look at [IBEIS](https://github.com/WildMeOrg/ibeis). 
+Developer contributions are very much appreciated. Refer to the [contribution guide](https://github.com/WildMeOrg/codex-frontend/blob/master/CONTRIBUTION_GUIDE.md). If you are a Python dev looking to help with our project, take a look at the [backend](https://github.com/WildMeOrg/houston). If you are a data scientist looking to help, take a look at [WB-IA](https://github.com/WildMeOrg/wildbook-ia).
```

Comment:
- WBIA isn't hyphenated in the WildBook IA README.

Reply:
Fixed.